src/scripts/OSFramework/OSUI/Pattern/AccordionItem/AccordionItemConfig.ts

```diff
@@ -9,48 +16,5 @@ namespace OSFramework.OSUI.Patterns.AccordionItem {
 		constructor(config: JSON) {
 			super(config);
 		}
-
-		/**
-		 * Method used to check if a given property can be changed!
-		 *
-		 * @param isBuilt if pattern is already built
-		 * @param key key name to be checked
-		 * @returns {boolean}
-		 * @memberof OSFramework.Patterns.AccordionItem.AccordionItemConfig
-		 */
-		public validateCanChange(isBuilt: boolean, key: string): boolean {
-			if (isBuilt) {
-				return key !== Enum.Properties.StartsExpanded;
-			}
-			return true;
-		}
-
-		/**
-		 * Method used to validate default value before apply it's change!
-		 *
-		 * @param key property name
-		 * @param value value to be set
-		 * @returns {*}
-		 * @memberof OSFramework.Patterns.AccordionItem.AccordionItemConfig
-		 */
-		public validateDefault(key: string, value: unknown): unknown {
-			let validatedValue = undefined;
-			switch (key) {
-				case Enum.Properties.IsDisabled:
-					validatedValue = this.validateBoolean(value as boolean, false);
-					break;
-				case Enum.Properties.Icon:
-					validatedValue = this.validateString(value as string, Enum.IconType.Caret);
-					break;
```

Comment:
@bmarcelino-fe I might be mistake, but I think we still need these. 

These types of custom validations, specially the ones based on entities are not covered by the abstract method. As well as others, that need to set a specific default value on a string or something.

The same might apply for all other changed Patterns on this PR

Reply:
Yes, we could. The behavior of that validation is in specific use cases and I didn't catched that. For thos patterns that has this type of validations to keep the defaults, we be rolled back.